metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/NetworkFunctions.java

```diff
@@ -224,7 +224,12 @@ private static InternetDomainName toDomainName(Object dnObj) {
     if(dnObj != null) {
       if(dnObj instanceof String) {
         String dn = dnObj.toString();
-        return InternetDomainName.from(dn);
+        try {
+          return InternetDomainName.from(dn);
```

Comment:
The modified configuration from your test that would test this is:

` /**
   {
    "fieldTransformations" : [
     {
       "transformation" : "STELLAR"
      ,"output" : [ "full_hostname", "domain_without_subdomains" ]
      ,"config" : {
         "full_hostname" : "URL_TO_HOST('http://1234567890123456789012345678901234567890123456789012345678901234567890/index.html')"
        ,"domain_without_subdomains" : "DOMAIN_REMOVE_SUBDOMAINS(full_hostname)"
                  }
     }
                      ]
   }
   */`